src/ostree/ot-admin-builtin-kargs.c

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (C) 2015 Colin Walters <walters@verbum.org>
```

Comment:
You probably want to add your name here too. Maybe we could trim down those headers too as we have SPDX identifiers already.

The date should also reflect the file creation date.